cmd/controller/main.go

```diff
@@ -53,6 +53,10 @@ func main() {
 		cfg.Burst = rest.DefaultBurst
 	}
 
+	// Multiply by number of controllers
```

Comment:
taskrun and pipelinerun controllers?

Reply:
Yes, Chains has two controllers - taskrun and pipelinerun.